stdlib/LinearAlgebra/src/symmetric.jl

```diff
@@ -254,11 +254,19 @@ end
     end
 end
 
+Base._reverse(A::Symmetric, dims::Integer) = (reverse((Matrix(A)); dims))
+Base._reverse(A::Symmetric, dims::Colon) = Symmetric(reverse(Matrix(A)))
```

Comment:
More efficient to just reverse `A.data` and then flip the `uplo`:
```suggestion
Base._reverse(A::Symmetric, dims::Colon) = Symmetric(reverse(A.data), A.uplo == 'U' ? :L : :U)
```